### DISCUSSION TITLE
📚 Share Your Favorite Books: Novels and Inspirational Reads! 🌟

### DISCUSSION THREAD

**Discussion by Ashmari** [NAIST-SE/AttractingContributors#1079] [2024 Attraction Contributions]
Hello Everyone!

Books are awesome—they can take us on wild adventures with gripping stories to distant lands or make us think about life in new ways. Let's make a list of our favorite books in two categories:
1. Novels: Share those books that kept you up late because you just had to know what happens next!
2. Inspirational Reads: Tell us about books that touched your heart or made you see things differently, offered valuable lessons, or reshaped your perspective.

Your recommendations will make our book collection super cool. Let's dive into the magic of reading together!

Please edit the ReadMe file and send a pull request.
Link: https://github.com/Ashmari/AttractingContributors_BookVoyage.git

**Comment by Canppchk** [comment]
I add Percy Jackson and the Olympians on your list.
could you help me with my project too? Thanks!!
https://github.com/Canppchk/AttractingContributors-Ramen

**Reply by Ashmari** [reply]
Thank you

**Reply by Canppchk** [reply]
i need 2 contributor left could you please help?

**Reply by Ashmari** [reply]
I contributed can you check?

**Reply by Canppchk** [reply]
I got it thanks!!

**Comment by mustafizur-r** [comment]
Hello [@Ashmari](https://github.com/Ashmari) , I recently made a contribution to your project , I have added "The Alchemist by Paulo Coelho " could you please contribute to my as well ? Following is the link to [my repository](https://github.com/MustafizurRU/AttractingContributors-My_Favorite_Sci-Fi_Books) . Thanks in advance.

**Reply by Ashmari** [reply]
Thank you, I have made the contribution by adding Digital Fortress and Da Vinci Code

**Comment by oyewaleoyebode** [comment]
Hello @Ashmari  , I recently made a contribution to your project.

**Reply by Ashmari** [reply]
Thank you.

**Comment by Drobotsan** [comment]
Hi, I contribute to your project.
Please accept my pull request.

Would you be kind to contribute with my project :D
https://github.com/Drobotsan/Places-in-the-world

Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1140

**Comment by Ryota-keda** [comment]
I just contribute to your project.
if you are ok, please contribute to my project too.
https://github.com/Ryota-keda/Baseball

**Comment by ShunHattori** [comment]
I just made a PR on your repo.
Could you contribute to my repo? Thanks in advance.
[https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git](https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git)

**Comment by anzhitingg** [comment]
Hello！
I have just made a PR on your repo. 
Could you contribute to mine? 
Here : https://github.com/anzhitingg/AttractingContributors-MBTI
Thank you very much！
